src/jest-types.ts

```diff
@@ -15,7 +15,20 @@ export type HasteConfig = {
   providesModuleNodeModules: Array<string>;
 };
 
-export type JestConfig = {
+export interface PostProcessorOptions {
+  plugins? : any[];
+  presets?: any[];
+  cacheDirectory? : string;
+  filename?: string;
+}
+
+export interface PostProcessHook {
+	(src: string, filename: string, config: JestConfig, transformOptions: TransformOptions) : string;
```

Comment:
I'm not opposed to the idea - this is just a rough draft. Does it make more sense to change the type now, or change it if/when we know what users want? I'm a little vary of overengineering.